### ISSUE TITLE
Feedback for “Getting Started with Bruno”

### ISSUE THREAD

**Issue by bbailey12** [usebruno/bruno-docs-legacy#71]
How do I use browser cookies as part of my collection?  I want to add a browser cookie but I can't figure out how to do it.  The docs don't give any detail.    See this PR:

https://github.com/usebruno/bruno/issues/968

**Comment by vsh263** [comment]
I know it's been a while but if anyone else runs into a similar issue, you could write a Script for your login or refresh request by going to the Scripts Tab then under Post Response, you can do the following:

```javascript
const cookieName = "refresh_token"; // Name of the cookie you'd like to extract
const cookiesHeader = res.headers["set-cookie"];

const findCookieValue = (headers, name) => {
  if (!Array.isArray(headers)) {
    return undefined;
  }
  const cookieRegex = new RegExp(`(?:^|;\\s*)${name}=([^;]+)`);
  const matchingCookie = headers.find(cookieStr => cookieRegex.test(cookieStr));
  if (matchingCookie) {
    const match = matchingCookie.match(cookieRegex);
    return match ? match[1] : undefined;
  }
  return undefined; 
}

const cookieValue = findCookieValue(cookiesHeader, cookieName);

console.log({ cookieValue });

bru.setVar(cookieName, cookieValue);
```
You can then use this variable for any request by passing `{{your_cookie_name}}` (in my case: `{{refresh_token}}`

**Tldr**: The script gets the "set-cookie" header from the response, then creates a [runtime variable](https://docs.usebruno.com/get-started/variables/runtime-variables) under the same name which can be accessed by all your requests in that collection.
